Update Watcher docs to reflect new states (#139435)
elastic/kibana: docs/management/watcher-ui/index.asciidoc

[float]
[[watcher-getting-started]]
=== View watch history and status

> The Watcher overview page lists your watches and each watch's state,

Reviewer: ```suggestion
The Watcher overview page lists your watches, including the state of each watch,
```

when its condition was last met, when the condition was last checked, and

> information about whether any of its actions have been acknowledged,

Reviewer: ```suggestion
whether any of its actions have been acknowledged,
```

throttled, or failed to execute. Each watch can be in one of three states:

* *Active.* The watch is working properly.

> * *Inactive.* The watch is working properly but has been deliberately disabled, so it won't execute any actions.

Reviewer: ```suggestion
* *Inactive.* The watch is working properly, but is deliberately disabled, so it won't execute any actions.
```

The *Simulate* tab allows you to override parts of the watch, and then run a
simulation. Be aware of these implementation details on overrides:

> * Trigger overrides use {ref}/common-options.html#date-math[date ma th].

Reviewer: This doesn't look right.

Author: Thanks for spotting that!
